### PR TITLE
Document `FFI\CType::getFuncParameterCount()` purpose and return

### PR DESCRIPTION
**Pull request by DanielEScherzer**

### CODE REVIEW

File: reference/ffi/ctype/getfuncparametercount.xml [hunk 1]
```diff
@@ -27,7 +25,9 @@
  <refsect1 role="returnvalues">
   &reftitle.returnvalues;
   <para>
-   
+   Returns the number of parameters for the underlying function type.
+	  If the underlying type is not a function, a
+	  <exceptionname>FFI\Exception</exceptionname> is thrown.
```
Comment (Girgias) [inline reference/ffi/ctype/getfuncparametercount.xml:30]: The indent is off here, we use 1 space for indentation for XML files.

**Review by Girgias — CHANGES_REQUESTED** [review]
Note the `<refpurpose>` tag still just says "Description"

File: reference/ffi/ctype/getfuncparametercount.xml (hunk 1 above)
Comment (DanielEScherzer) [reply reference/ffi/ctype/getfuncparametercount.xml:30]: This entire block is indented 1 space more than than the `<para>` tag, is that not correct?

**Comment by DanielEScherzer** [comment]
> Note the `<refpurpose>` tag still just says "Description"

Done

File: reference/ffi/ctype/getfuncparametercount.xml (hunk 1 above)
Comment (cmb69) [reply reference/ffi/ctype/getfuncparametercount.xml:30]: There is a tab at the beginning of line 29 and 30.

Comment (DanielEScherzer) [reply reference/ffi/ctype/getfuncparametercount.xml]: Ah, thanks

File: reference/ffi/ctype/getfuncparametercount.xml [hunk 2]
```diff
@@ -27,7 +25,9 @@
  <refsect1 role="returnvalues">
   &reftitle.returnvalues;
   <para>
-   
+   Returns the number of parameters for the underlying function type.
+   If the underlying type is not a function, a
+   <exceptionname>FFI\Exception</exceptionname> is thrown.
```
Comment (cmb69) [inline reference/ffi/ctype/getfuncparametercount.xml:30]: That should probably be
```suggestion
   If the underlying type is not a function, an
   <exceptionname>FFI\Exception</exceptionname> is thrown.
```
(at least I read FFI as EFF-EFF-I)

Comment (DanielEScherzer) [reply reference/ffi/ctype/getfuncparametercount.xml]: done

**Review by cmb69 — APPROVED** [review]
Thank you! This looks fine to me.

**Review by Girgias — APPROVED** [review]
Thank you!

**Pull request merged**